src/main/java/core/basesyntax/service/impl/FileOperationImpl.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.dto.FruitRecordDto;
+import core.basesyntax.service.FruitOperationHandler;
+import core.basesyntax.service.FruitRecordDtoParser;
+import core.basesyntax.service.OperationStrategy;
+import java.util.List;
+import java.util.Map;
+
+public class FileOperationImpl implements OperationStrategy {
+    private static final String LINE_SEPARATOR = ",";
```

Comment:
```suggestion
    private static final String CSV_SEPARATOR = ",";
```
usually by line separator people mean "\n"